bubuku/zookeeper/exhibior.py

```diff
@@ -3,23 +3,22 @@
 
```

Comment:
I just noticed that the name of this file contains a typo. It might be worth renaming once all PRs are merged.